Platform/QemuBoardPkg/BoardConfig.py

```diff
@@ -145,6 +146,21 @@ def GetDscLibrarys (self):
 		]
 		return dsc_libs
 
+	def GetContainerList (self):
+		container_list = []
+		container_list.append ([
+		  # Name       | Image File |    CompressAlg  | AuthType   | Key File                     | Region Size
+		  # =====================================================================================================================
+		  ('IPFW',      'SIIPFW.bin',    '',           'RSA2048',   'TestSigningPrivateKey.pem',    0     ),   # Container Header
+		  ('TST1',      '',              'Dummy',      '',          '',                             0x2000),   # Component 1
+		  ('TST2',      '',              'Lz4',        '',          '',                             0x3000),   # Component 2
```

Comment:
Yes, we do.  Here, since we don't have additional binary files checked in the repo,  I just leave it blank.  It means the space will be reserved for TST2.  Later on , people can replace valid data into this TST2 region using the GenContainer tool.

Reply:
great. thanks for confirmation.